mobile_back_qti/cpp/backend_qti/settings/qti_settings_default_cpu.pbtxt

```diff
@@ -31,16 +31,15 @@ benchmark_setting {
     delegate_name: "SNPE_CPU"
     accelerator_name: "snpe_cpu"
     accelerator_desc: "CPU"
-    model_path: "https://github.com/mlcommons/mobile_models/releases/download/v3.0-snpe/mobilenet_edgetpu_224_1.0_htp.dlc"
-    model_checksum: "2e7c4d33b480b5566bdf05e1204b6152"
+    model_path: "local:///mlperf_models/mobilenet_edgetpu_224_1.0_htp.dlc"
```

Comment:
Please update all local path to URL from
https://github.com/mlcommons/mobile_models/releases/tag/v3.1-snpe